packages/cardano-services/src/ChainHistory/DbSyncChainHistory/queries.ts

```diff
@@ -25,8 +25,10 @@ const selectTxOutput = (collateral = false) => `
 		tx_out.value AS coin_value,
 		tx_out.data_hash AS datum,
 		tx_out.reference_script_id as reference_script_id,
+		bytes,
 		tx.hash AS tx_id
 	FROM ${collateral ? 'collateral_tx_out' : 'tx_out'} AS tx_out 
+	LEFT JOIN datum ON datum.id = inline_datum_id
```

Comment:
`bytes` is not an ambiguous column (otherwise the query would fail): there is no need to specify the source table.
Specifying the source table when it's not required would only (slightly) increases the DB workload to parse the query.

Reply:
It's just a readability improvement. If it was up to me, i'd say queries with multiple tables should always have explicitly defined select clause.

Load change will not be noticeable at all

Reply:
bcb57529015 should solve this